kratos/containers/array_1d.h

```diff
@@ -70,19 +70,19 @@ class	array_1d	: public boost::numeric::ublas::vector_expression< array_1d<T, N>
     /// Pointer definition of	array_1d
     KRATOS_CLASS_POINTER_DEFINITION(array_1d);
 
-    typedef std::size_t size_type;
-    typedef	std::ptrdiff_t difference_type;
-    typedef	T value_type;
-    typedef	typename boost::numeric::ublas::type_traits<T>::const_reference const_reference;
-    typedef	T &reference;
-    typedef	std::array<T,N> array_type;
-    typedef	T *pointer;
-    typedef	array_1d<T, N> self_type;
-    typedef	const boost::numeric::ublas::vector_reference<const self_type>	const_closure_type;
-    typedef	boost::numeric::ublas::vector_reference<self_type>	closure_type;
-    typedef	self_type vector_temporary_type;
-    typedef	boost::numeric::ublas::dense_tag storage_category;
-//		typedef	concrete_tag simd_category; //removed for the new ublas
+    using size_type = std::size_t;
+    using difference_type = std::ptrdiff_t;
+    using value_type = T;
+    using const_reference = typename boost::numeric::ublas::type_traits<T>::const_reference;
+    using reference = T&;
```

Comment:
Pretty sure this is wrong, even if it compiles, it changes the meaning

Reply:
ok its correct

Reply:
Okay